editor/scene_tree_dock.cpp

```diff
@@ -117,6 +131,8 @@ void SceneTreeDock::_reset_hovering_timer() {
 	if (!inspect_hovered_node_delay->is_stopped()) {
 		inspect_hovered_node_delay->stop();
 	}
+	_restore_treeitem_custom_color(tree_item_inspected);
+	tree_item_inspected = nullptr;
```

Comment:
I don't think this is needed anymore since your last push

Reply:
I think you are right